public/components/metrics/index.tsx

```diff
@@ -116,7 +117,7 @@ export const Home = ({
 
   useEffect(() => {
     if (!editMode) {
-      selectedMetrics.length > 0 ? setIsTopPanelDisabled(false) : setIsTopPanelDisabled(true);
+      setIsTopPanelDisabled(!(selectedMetrics.length > 0));
```

Comment:
eslint forced me to make this change - it is logically equivalent ?

Reply:
looks good to me. The prior one is more readable though. An easy way to check this is to use the edit button can check manually.

Reply:
Uhh.. based on the terminology :
`setIsTopPanelDisabled(selectedMetrics.length == 0);`

@ps48 would that be readable enough?

PS: Tertiary conditions are almost always less readable.  This sentiment is > IMHO